### ISSUE TITLE
reportUninitializedInstanceVariable false positives for typing.Protocol

### ISSUE THREAD

**Issue by ovsds** [microsoft/pyright#3746]
Class attributes in `typing.Protocols` trigger false positive `reportUninitializedInstanceVariable`.

Trying different strategies as seen in the example below:
``` python
class MyProtocol(typing.Protocol):
    x: str = ...
    y: str
```
Both approaches spawn errors:
```
 error: Expression of type "ellipsis" cannot be assigned to declared type "str"
    "ellipsis" is incompatible with "str" (reportGeneralTypeIssues) 
 warning: Instance variable "y" is not initialized in the class body or __init__ method (reportUninitializedInstanceVariable)
```
First I kinda agree with, but the second seems to be false positive.

The only working workaround found is:
``` python
class MyProtocol(typing.Protocol):
    @property
    def x(self) -> str:
        ... 
```
but it kinda feels too excessive when describing config protocols with plenty of fields.

Thanks for your work! :)

**Comment by evgenymarkov** [comment]
<img width="352" alt="Screenshot 2022-07-26 at 13 59 18" src="https://user-images.githubusercontent.com/16443248/180990770-ed859d0a-1348-4cc7-9c1f-3a52bab07368.png">
<img width="1103" alt="Screenshot 2022-07-26 at 13 59 12" src="https://user-images.githubusercontent.com/16443248/180990786-a42f1b00-ccde-46f7-a987-c07095e97574.png">

**Comment by erictraut** [comment]
I agree that this check should not apply to protocol classes. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.264, which I just published. It will also be included in a future release of pylance.
